godot-core/src/meta/error/call_error.rs

```diff
@@ -396,7 +419,9 @@ enum SourceError {
         erased_error: ErasedConvertError,
         value: String,
     },
-    Call(Box<CallError>),
+
+    // Used to be Box<CallError>, but now CallError itself is boxed.
```

Comment:
Do we need this info?

Reply:
Yes, because if ever we unbox it, we'll have a cyclic dependency. This is the place where we would need to reintroduce the `Box`.

Reply:
Reworded it to be a bit clearer.